### PR TITLE
[PPD-97] CU - Generate output csv

### PR DESCRIPTION
**Pull request by pasqualespica**
The purpose of this PR is to add implementation of `CuGenerateOutputCsv` Az trigger timer fn
image

#### List of Changes

Add  : 
  - `CuGenerateOutputCsv` fn
  - `DebtPositionService` service

#### Motivation and Context

[Canone Unico prj](https://pagopa.atlassian.net/wiki/spaces/PCU/pages/456032840/Design+Review)

#### How Has This Been Tested?

junit

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: �6�`,J [hunk 1]
```diff
@@ -0,0 +1,3 @@
+id,pa_id_istat,pa_id_catasto,pa_id_fiscal_code,pa_id_cbill,pa_pec_email,pa_referent_email,pa_referent_name,amount,debtor_id_fiscal_code,debtor_name,debtor_email,payment_notice_number,note
```
Comment (jacopocarlini) [inline �6�`,J:1]: 🤔  the name of this file is: �6�`,J

File: pom.xml [hunk 2]
```diff
@@ -50,35 +51,40 @@
...
+        </dependency>
         <!-- Azure END-->
 
         <!-- Test START-->
 
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter</artifactId>
-            <version>5.8.2</version>
+            <version>5.4.2</version>
```
Comment (jacopocarlini) [inline pom.xml:66]: check the versions

**Review by jacopocarlini — CHANGES_REQUESTED** [review]

File: �6�`,J (hunk 1 above)
Comment (pasqualespica) [reply �6�`,J:1]: right, tnx

File: pom.xml (hunk 2 above)
Comment (pasqualespica) [reply pom.xml:66]: tnx, i'll fix it

File: pom.xml [hunk 3]
```diff
@@ -204,6 +212,7 @@
                 <version>0.8.7</version>
                 <configuration>
                     <excludes>
+                        <exclude>**/config/*</exclude>
```
Comment (jacopocarlini) [inline pom.xml:215]:
```suggestion
```

**Review by jacopocarlini — APPROVED** [review]
great 🏆

**Pull request merged**